waspc/README.md

```diff
@@ -214,52 +199,16 @@ On any changes you do to the source code of Wasp, Wasp project gets recompiled,
 - data/Generator/templates/ -> mustache templates for the generated client/server.
 - examples/ -> example apps
 
-## Building / development (detailed)
-
-Some useful cabal commands:
-
-- `cabal update` to update your package information.
-- `cabal build` to build the project, including `wasp` binary which is both CLI and compiler in one.
-- `cabal run wasp-cli <arguments>` to run the `wasp` binary that was previously built.
-- `cabal test` to build the whole project + tests and then also run tests.
-- `cabal install` -> builds the project and places the binary so it is in PATH (so you can call it directly, from anywhere, with just `wasp`).
-
-For live compilation and error checking of your code we recommend using Haskell Language Server (hls) via your IDE, but if that is not working as it should, then safe fallback is always `ghcid`. You can install `ghcid` globally with `cabal install ghcid` and then just type `ghcid` when in the project -> it will watch for any file changes and report errors.
-
-### Run script
```

Comment:
This information still exists, but it got moved much sooner in this doc, in "Basics" where we introduce `run` script. I also shortened it a bit, without loosing anything I believe.